.cargo/config.toml

```diff
@@ -0,0 +1,32 @@
+#
+# An auto defined `clippy` feature was introduced,
+# but it was found to clash with user defined features,
+# so was renamed to `cargo-clippy`.
+#
```

Comment:
nit: Interesting history, but unrelated to our codebase, I think this is unneeded.